emailApp/untitled

```diff
@@ -0,0 +1,10 @@
+	 		 	time_difference = sgt_now() - cms.last_sent
+	 	time_difference_in_minutes = time_difference / timedelta(minutes=1)
+
+	 	if (cms.active ==True and time_difference_in_minutes>=30):
```

Comment:
If I remember, Python is nice in that it can be written to make it very readable. So this can be rewritten as:
```
if cms.active and time_difference_in_minutes >= 30:
   # stuff
```
Why I like Python!